cmd/storj-sim/network.go

```diff
@@ -154,6 +154,7 @@ func networkTest(flags *Flags, command string, args []string) error {
 	if printCommands {
 		fmt.Fprintf(processes.Output, "exec: %v\n", strings.Join(cmd.Args, " "))
 	}
+	time.Sleep(6 * time.Second) //hack: this is so the contact chore can send the satellite the capacity info on its second iteration after 5s
```

Comment:
do you know what kind of impact this has on jenkins performance?

Reply:
Doesn't look like it makes much of a difference (crossed referenced some other prs)

Reply:
The impact is that it can make things flaky, making Jenkins failures less understandable. So, we need to figure out a better approach.

Reply:
@jenlij wrote a ticket to fix this longer term:  https://storjlabs.atlassian.net/browse/V3-2862